builder/Makefile.tracee-container

```diff
@@ -105,7 +105,7 @@ build-tracee: \
 	$(CMD_DOCKER) images \
 		--filter "label=AS=tracee-make" \
 		--format "{{.ID}}" \
-		| xargs $(CMD_DOCKER) rmi
+		| xargs -r $(CMD_DOCKER) rmi
```

Comment:
Actually the idea here was to save space:

![image](https://user-images.githubusercontent.com/7395852/223437697-777d97e9-d3e2-4539-9ee3-5d0cbc0aa39b.png)

by removing the "tracee-make" container image. But the image is named "ubuntu-tracee-make" and the docker filter is not picking it.

Reply:
Actually, the `Makefile.tracee-make" script generates this container. I believe, the `Makefile.tracee-container` script should not be responsible to clean up after the first one.

Reply:
But this tracee-make container used for the container building process and shouldn't be used by anything else. Why do you think we shouldn't clean it ?

You can have the "tracee-make" container using:

```
$ make -f builder/Makefile.tracee-make help

To create an alpine-tracee-make container:

    $ make -f builder/Makefile.tracee-make alpine-prepare

To create an ubuntu-tracee-make container:

    $ make -f builder/Makefile.tracee-make ubuntu-prepare

To execute an alpine-tracee-make shell:

    $ make -f builder/Makefile.tracee-make alpine-shell

To execute an ubuntu-tracee-make shell:

    $ make -f builder/Makefile.tracee-make ubuntu-shell

Examples:

    $ make -f builder/Makefile.tracee-make ubuntu-make ARG="clean"
    $ make -f builder/Makefile.tracee-make ubuntu-make ARG="bpf-core"
    $ make -f builder/Makefile.tracee-make ubuntu-make ARG="tracee-ebpf"

Tell tracee-make to do STATIC builds:

    $ STATIC=0 make -f builder/Makefile.tracee-make alpine-make ARG="tracee-ebpf"
    $ STATIC=1 make -f builder/Makefile.tracee-make alpine-make ARG="tracee-ebpf"
```